ops/scripts/geth.sh

```diff
@@ -38,4 +29,16 @@ curl \
     --retry-delay 1 \
     $ROLLUP_CLIENT_HTTP
 
-exec geth --verbosity="$VERBOSITY" "$@"
+# import the private key
+echo "Importing private key"
+echo "6587ae678cf4fc9a33000cdbf9f35226b71dcc6a4684a31203241f9bcfd55d27" > key.prv
```

Comment:
TODO: make this an environment variable so we don't have to hardcode it everywhere